src/main/java/org/folio/consortia/service/impl/PublicationServiceImpl.java

```diff
@@ -162,53 +179,51 @@ private PublicationTenantRequestEntity savePublicationTenantRequest(PublicationT
     }
```

Comment:
it looks like we're setting context with same content as it was, could be removed